apps/voting-disputable/app/src/components/VoteDetail/DisputableActionStatus.js

```diff
@@ -60,26 +65,14 @@ function DisputableActionStatus({ vote }) {
             </span>
           </div>
         </Item>
-        <Item label="Challenge period">
-          {new Date().getTime() > vote.disputable.action.endDate ? (
-            formatDate(vote.disputable.action.endDate)
-          ) : (
-            <div
-              css={`
-                display: inline-flex;
-              `}
-            >
-              <Timer end={new Date(vote.disputable.action.endDate)} />{' '}
-              <div
-                css={`
-                  padding-left: ${1 * GU}px;
-                  color: ${theme.contentSecondary};
-                `}
-              >
-                (48h)
-              </div>
-            </div>
-          )}
+        <Item label={challenged ? 'Settlement period' : 'Challenge period'}>
+          <DisputablePeriod
+            startDate={
+              challenged
+                ? vote.disputable.pausedAt
+                : new Date(vote.data.startDate).getTime()
+            }
+          />
```

Comment:
Nice work encapsulating this! don't forgot to remove any redundant imports in this file such as `theme`, `timer`, `format` :)